### ISSUE TITLE
AutoTool

### ISSUE THREAD

**Issue by SusgUY446** [MeteorDevelopment/meteor-client#4584]
### Describe the bug

AutoTool is not working

### Steps to reproduce

Using the stable (1.20.4) version and activating AutoTool (default settings)

### Meteor Version

0.5.6

### Minecraft Version

1.20.4

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [x] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by 19MisterX98** [comment]
cannot reproduce.
Just so that there is no confusion. You are holding the correct tool to mine a block in another hotbar slot and auto tool does not switch to that slot when you try to mine the block?

**Comment by SusgUY446** [comment]
yes. i have a diamond pickaxe. but when i click the block it uses the current tool (shovel)

**Comment by Gingerbeard5773** [comment]
cant reproduce

**Comment by susgirl446** [comment]
its on a outdated version anyway
